keycloak-plugin/sasl-plugin/pom.xml

```diff
@@ -29,6 +29,26 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>openshift-client</artifactId>
```

Comment:
I think it might be worth using just okhttp client or some other http client already part of keycloak to perform the requests. The token can be dug up from `/var/run/secrets/kubernetes.io/serviceaccount/token`

Reply:
Is it just a matter of adding some headers to the HTTP request with the token for authorisation?

Reply:
Yes, this is done in the doRawHttpRequest method:

             .addHeader("Authorization", "Bearer " + client.getConfiguration().getOauthToken())

Reply:
Given we include it elsewhere, I'm minded to keep as is